src/components/Login.js

```diff
@@ -80,7 +80,6 @@ class Login extends Component {
                     </li>
                   </ul>
                 </div>
-              </div>
             </nav>
             <div className="wrapper wrapper-full-page">
               <div className="full-page login-page full-page-background" filter-color="blue" data-image="/assets/img/spacex.jpg" style={{backgroundImage: `url(/assets/img/spacex.jpg)`, width: `100%`}}>
```

Comment:
on line 98 - there's an apostrophe which breaks the syntax highlighting. you can replace it with an ASCII, so that the syntax highlighting still works.